scripts/scil_outlier_rejection.py

```diff
@@ -11,34 +11,32 @@
 import argparse
 import logging
 
-import nibabel as nib
-from nibabel.streamlines.tractogram import Tractogram
-import numpy as np
+from dipy.io.streamline import save_tractogram
 
+from scilpy.io.streamlines import load_tractogram_with_reference
 from scilpy.io.utils import (add_overwrite_arg,
+                             add_reference_arg,
                              assert_inputs_exist,
                              assert_outputs_exist,
                              check_tracts_same_format)
-from scilpy.tractanalysis.features import (
-    outliers_removal_using_hierarchical_quickbundles,
-    prune)
+from scilpy.tractanalysis.features import remove_outliers
 
 
 def _build_arg_parser():
-    parser = argparse.ArgumentParser(
-        description=__doc__,
-        formatter_class=argparse.RawTextHelpFormatter)
-    parser.add_argument('in_bundle',
-                        help='Fiber bundle file to remove outliers from.')
-    parser.add_argument('out_bundle',
-                        help='Fiber bundle without outliers.')
-    parser.add_argument('--remaining_bundle',
-                        help='Removed outliers.')
-    parser.add_argument('--alpha', type=float, default=0.6,
-                        help='Percent of the length of the tree that clusters '
-                             'of individual streamlines will be pruned.')
-    add_overwrite_arg(parser)
-    return parser
+    p = argparse.ArgumentParser(description=__doc__,
+                                formatter_class=argparse.RawTextHelpFormatter)
+    p.add_argument('in_bundle',
+                   help='Fiber bundle file to remove outliers from.')
+    p.add_argument('out_bundle',
+                   help='Fiber bundle without outliers.')
+    p.add_argument('--remaining_bundle',
+                   help='Removed outliers.')
+    p.add_argument('--alpha', type=float, default=0.6,
+                   help='Percent of the length of the tree that clusters '
+                   'of individual streamlines will be pruned.')
+    add_reference_arg(p)
+    add_overwrite_arg(p)
+    return p
```

Comment:
Why you don't have the fast option here ?

Reply:
Because we are more patient, 1-50 bundle that makes sense to be slow and reliable.
decompose is more like 3000-25000 that no one QA, so an approx is ok